app/views/email_confirmation/_follow_up.slim

```diff
@@ -1 +1,15 @@
-= @rendered
+.header-logo.header-logo--light
```

Comment:
just a little confused why this is in a partial, and not directly a part of the view.